### ISSUE TITLE
[BUG] gam info resoldcustomer <customerId> [formatjson] inconsistent behaviour 

### ISSUE THREAD

**Issue by jay-eleven** [taers232c/GAMADV-XTD3#348]
Hi Ross!

While playing around the `gam info resoldcustomer` command I noticed a minor inconsistency.

Given the following customer contact information:
![image](https://user-images.githubusercontent.com/46608557/227194263-f0b5e1bb-a2ff-4510-8441-db721a2dc7ee.png)
First, without `formatjson`:
```
jay@cloudshell:~$ gam info resoldcustomer C0xxxxxxx
Customer ID: C0xxxxxxx
Customer Domain: domain.com
...
Phone: +1 4158000000
Customer Alternate Email:
```
The primary admin email (**admin@domain.com**) is missing. 

Now with formatjson:
```
jay@cloudshell:~$ gam info resoldcustomer C0xxxxxxx formatjson
{"alternateEmail": "", 
"customerDomain": "domain.com", 
... , 
"customerId": "C0xxxxxxx", 
"customerType": "team", 
"phoneNumber": "+1 4158000000", 
... , 
"primaryAdmin": {"primaryEmail": "admin@domain.com"}, 
...
```
The primary admin email (**admin@domain.com**) is shown.

Nothing too serious.

**Comment by taers232c** [comment]
Thanks, will fix this morning.

Ross
----
Ross Scroggs
***@***.***

**Comment by taers232c** [comment]
Fixed in 6.52.03

Thanks

**Comment by jay-eleven** [comment]
Fixed!

Thanks Ross.
